v1models/transform.go

```diff
@@ -33,6 +38,13 @@ func TransformProfileFromV2ToV1(profile models.DeviceProfile) (DeviceProfile, er
 	v1dp.DeviceCommands = toV1DeviceCommands(profile.DeviceCommands)
 	v1dp.CoreCommands = toV1CoreCommand(profile.DeviceResources, profile.DeviceCommands)
 
+	if isModbusAttributes(v1dp.DeviceResources[0].Attributes) {
```

Comment:
don't hardcode the index here.  it would cause nil pointer panic if DeviceResources is an empty array

Reply:
Modified. Added code to check slice lenght.